From c4c6e31f49fecbe8586e0169da31860adcbc935f Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Wed, 30 Nov 2016 14:58:32 -0800
Subject: [PATCH 07/10] iscsi: fix hp hba

Adjust some options so LIO can talk to the HP iSCSI HBA
---
 drivers/target/iscsi/iscsi_target_parameters.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/target/iscsi/iscsi_target_parameters.c b/drivers/target/iscsi/iscsi_target_parameters.c
index 8187262..de0ef95 100644
--- a/drivers/target/iscsi/iscsi_target_parameters.c
+++ b/drivers/target/iscsi/iscsi_target_parameters.c
@@ -482,10 +482,12 @@ int iscsi_set_keys_to_negotiate(
 			SET_PSTATE_NEGOTIATE(param);
 		} else if (!strcmp(param->name, FIRSTBURSTLENGTH)) {
 			SET_PSTATE_NEGOTIATE(param);
+/* NETGEAR *
 		} else if (!strcmp(param->name, DEFAULTTIME2WAIT)) {
 			SET_PSTATE_NEGOTIATE(param);
 		} else if (!strcmp(param->name, DEFAULTTIME2RETAIN)) {
 			SET_PSTATE_NEGOTIATE(param);
+ * NETGEAR */
 		} else if (!strcmp(param->name, MAXOUTSTANDINGR2T)) {
 			SET_PSTATE_NEGOTIATE(param);
 		} else if (!strcmp(param->name, DATAPDUINORDER)) {
-- 
1.9.1

